packages/lesswrong/components/ultraFeed/FeedContentBody.tsx

```diff
@@ -10,17 +10,12 @@ import { defineStyles, useStyles } from '../../components/hooks/useStyles';
 const styles = defineStyles('FeedContentBody', (theme: ThemeType) => ({
   root: {
     position: 'relative',
-    '& .read-more-button': {
+    '& .read-more-suffix': {
```

Comment:
Any reason to change this classname?

Reply:
it's not a button anymore (to the extent it ever way)